src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -85,7 +85,7 @@ struct dfs_mt {
 
 	_Atomic uint32_t     inited;
 	char                *pool, *cont;
-	char                 fs_root[DFS_MAX_PATH];
+	char                *fs_root;
```

Comment:
I assume discover_daos_mount() needs updating for this as well?

Reply:
Fixed. Thank you!